### PR TITLE
Expand vars in tag and name inputs

### LINKED ISSUE #1048: $RESOLVED_VERSION not expanded when set via action input
Using the `name` input:
```yml
...
jobs:
  update_release_draft:
    runs-on: ubuntu-latest
    steps:
      - uses: release-drafter/release-drafter@v5
        with:
          name: "$RESOLVED_VERSION"
...
```
the release draft will be named `$RESOLVED_VERSION`, which obviously is not what I expected.

However inside the config file
```
name-template: '$RESOLVED_VERSION'
```
the release draft will correctly be named with the version, e.g. `1.0.0`.

Now, the `name` overrides the `name-template`:
<kbd></kbd>

I expect both configs behave identically, I would have guessed variable expansion takes places after the override happened, so it shouldn't matter where the config string came from.

### PR DESCRIPTION
**Pull request by mkurz**
Fixes #1048

### CODE REVIEW

**Review by jetersen — APPROVED** [review]
Looks good to me.
Could you add a test.

You should be able to find a mention of `GITHUB_ACTION` inside `index.test.js` via search.

**Comment by mkurz** [comment]
@jetersen Done. Tests added (after some problems do get things running :wink:). Can you merge?

File: test/fixtures/config/config-with-name-and-tag-template.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+template: Placeholder with example. Automatically calculated values based on previous releases are next major=$NEXT_MAJOR_VERSION, minor=$NEXT_MINOR_VERSION, patch=$NEXT_PATCH_VERSION.
+name-template: 'v1.2.3 (Code name: Hello World)'
+tag-template: v3.4.5
```
Comment (jetersen) [inline test/fixtures/config/config-with-name-and-tag-template.yml:3]: ? Could we use some $RESOLVED_VERSION so at least the test looks sane 😓 I cannot handle that your getting one version in name and a different in tag.

**Review by jetersen — CHANGES_REQUESTED** [review]
😓 Test output looks odd, would prefer the config was aligned and the test overrides the name or tag properly.

**Comment by mkurz** [comment]
Better? What I want to show is that once the tag gets overriden, once the name.

**Review by jetersen — APPROVED** [review]

**Comment by jetersen** [comment]
> Better? What I want to show is that once the tag gets overriden, once the name.

Much 😅 The tests clearly shows what your looking for 🙂

**Comment by jetersen** [comment]
Thank you for your contribution 🙌

**Comment by mkurz** [comment]
Thank you!

**Pull request merged**